lib/IRGen/GenExistential.cpp

```diff
@@ -842,8 +842,13 @@ class OpaqueExistentialTypeInfo final :
     // Use copy-on-write existentials?
     auto fn = getAssignBoxedOpaqueExistentialBufferFunction(
         IGF.IGM, getLayout(), objPtrTy);
-    auto call =
-        IGF.Builder.CreateCall(fn, {dest.getAddress(), src.getAddress()});
+    auto call = IGF.Builder.CreateCall(
```

Comment:
Should these bitcasts be explicitly sequenced outside the construction of this object?

Reply:
Damn it!  You would think that I would learn this based on the fact that I'm fixing this exact issue.  You're right @CodaFi.